src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -4130,7 +4140,7 @@ protected PaddingTermination getPaddingTermination()
      * @return the <tt>RemoteBitrateEstimator</tt> of this
      * <tt>VideoMediaStream</tt> if any; otherwise, <tt>null</tt>
      */
-    public RemoteBitrateEstimator getRemoteBitrateEstimator()
+    public RemoteBitrateEstimatorWrapper getRemoteBitrateEstimator()
```

Comment:
should the name of this method change to `getRemoteBitrateEstimatorWrapper() to match`?  btw, i don't care too much if we have a separate type for the wrapper or if we continue to call it `RemoteBitrateEstimator` and just have it contain an inner delegate that does the real estimation.

Reply:
can this just return the `RemoteBitrateEstimator` type?

Reply:
No, we can't return a `RemoteBitrateEstimator` because of the above two comments; but I can rename the method if this is your suggestion.